composer/loggers/mlflow_logger.py

```diff
@@ -60,6 +60,10 @@ class MLFlowLogger(LoggerDestination):
             log system metrics (CPU/GPU/memory/network usage) during training. (default: ``True``)
         ignore_metrics (List[str], optional): A list of glob patterns for metrics to ignore when logging. (default: ``None``)
         ignore_hyperparameters (List[str], optional): A list of glob patterns for hyperparameters to ignore when logging. (default: ``None``)
+        run_group (str, optional): A string to group runs together. (default: ``None``)
+        resume (bool, optional): If ``True``, Composer will search for an existing run tagged with
+            the `run_name` and resume it. If ``False``, Composer will create a new run. (default:
+            ``False``)
```

Comment:
Can we add something in docstring that says if True but no run is found, a new one will be created?

Reply:
sure, done!